src/components/button/index.tsx

```diff
@@ -8,10 +8,16 @@ export const Button = ({
   variant = "default",
   onClick,
   children,
+  width = "auto",
   ...props
 }: ButtonProps) => {
   return (
-    <ButtonContainer onClick={onClick} variant={variant} {...props}>
+    <ButtonContainer
+      onClick={onClick}
+      variant={variant}
+      width={width}
+      {...props}
+    >
       {children}
     </ButtonContainer>
   );
```

Comment:
you should create other branch for fix button

Reply:
This customization wasn't needed until creating navbar component - that's why it's in this pr and this branch